docs/concepts/apiserver-admission-plugins.md

```diff
@@ -107,7 +107,7 @@ It ensures that the finalizers of these resources are not removed by users, as l
 For `CredentialsBinding`s and `SecretBinding`s this means, that the `gardener` finalizer can only be removed if the binding is not referenced by any `Shoot`.
 In case of `Shoot`s, the `gardener` finalizer can only be removed if the last operation of the `Shoot` indicates a successful deletion. 
 
-## `ProjectValidator`
+## `ProjectMutator`
```

Comment:
Could we change the release note from

```
The `ProjectValidator` Admission plugin is now named `ProjectMutator`
```

to

```
The `ProjectValidator` admission plugin is now renamed `ProjectMutator`. If you have references to the old name of the admission plugin, make sure to adapt them before upgrading to this version of Gardener.
```

?

Describing the breaking change is great. We could add a sentence on what the operator has to do to adapt to the breaking change when upgrading to corresponding version of Gardener.